src/_P144_Vindriktning.ino

```diff
@@ -0,0 +1,376 @@
+//#######################################################################################################
+//################################## Plugin 144: Dust Sensor Ikea Vindriktning ##########################
+//#######################################################################################################
+/*
+  Plugin is based upon various open sources on the internet. Plugin uses the serial lib to access a serial port
+  This plugin was written by Mark Flesch
+
+  This plugin reads the particle concentration from the PM1006 sensor in the Ikea Vindriktning
+  The original Ikea processor is controlling the PM1006 sensor, the plugin is eavesdropping the responses
+  DevicePin1 - RX on ESP, TX on PM1006
+  DevicePin2 - TX on ESP, RX on PM1006 (optional, currently not accessed by the plugin)
+*/
+
+
+// #include section
+#include "_Plugin_Helper.h"
+#ifdef USES_P144
+# include <ESPeasySerial.h>
+
+// Standard plugin defines
+#define PLUGIN_144
+#define PLUGIN_ID_144     144                   // plugin id
+#define PLUGIN_NAME_144   "Dust - Vindriktning" // "Plugin Name" is what will be dislpayed in the selection list
```

Comment:
When adding support for a standalone PM1006 and/or PM1006K, the name of the plugin could better be changed to `Dust - PM1006(K) (Ikea Vindriktning)` or similar, not everybody will recognize the Vindriktning moniker, also I'm thinking a few years ahead, when possibly the product either gets renamed or no longer exists.